src/pages/AddPersonalBankAccountPage.js

```diff
@@ -163,7 +119,14 @@ class AddPersonalBankAccountPage extends React.Component {
                         </FixedFooter>
                     </>
                 ) : (
-                    <FormScrollView>
+                    <Form
+                        formID={ONYXKEYS.PERSONAL_BANK_ACCOUNT}
+                        isSubmitButtonVisible={!_.isUndefined(this.state.selectedPlaidBankAccount)}
+                        submitButtonText={this.props.translate('common.saveAndContinue')}
+                        onSubmit={this.submit}
+                        validate={this.validate}
+                        style={[styles.mh5, styles.mb5, styles.flex1]}
+                    >
                         <View style={[styles.mh5, styles.mb5, styles.flex1]}>
```

Comment:
This View is duplicating the styles applied by Form. I think we should just make it a fragment.
```suggestion
                        <>
```